analytics/src/main/java/com/bytegain/analytics/ConnectionFactory.java

```diff
@@ -36,8 +45,9 @@ public HttpURLConnection projectSettings(String writeKey) throws IOException {
    */
   public HttpURLConnection upload(String writeKey) throws IOException {
     // 10.0.2.2 is emulator's route to host's 127.0.0.1
+    final String localUrl = String.format(Locale.US, "http://10.0.2.2:%d/v1/batch", localServerPort);
```

Comment:
WHy no tmake the who URL a config param?

Reply:
I haven't needed it yet.  We could make it the full URL if we want to be able to set up a test server (staging?) for customers.

Should I change the iOS version too?  It's just a port.

Reply:
Fine for now, just wondering if there was something specific driving this